en/discover/portal/articles/100-web-experience-management/07-staging-content-for-publication/03-using-the-staging-environment/02-advanced-publication-with-staging.markdown

```diff
@@ -78,8 +78,9 @@ Content label. Otherwise, the Web Content section is absent.
 The *Categories* content type is not dependent on the date range and is always
 shown in the list.
 
-| **Note:** Since comments and ratings are meant for the end user, they are not
-| supported in staging and can only be added to the live site.
+| **Note:** Since some content types are meant for the end user (e.g. comments,
+| ratings, and custom fields), they are not supported in staging and can only be
+| added to the live site.
```

Comment:
Hey @Claudioss23 , I don't think this is exactly what the ticket is asking. The tickets wants us to mention that once published, **custom fields do not get deleted in remote live**. 

We could maybe work with `Note: Since comments and ratings are meant for the end user, they are not supported in staging and can only be added to the live site. Additionally, once published, custom fields do not get deleted in remote live.`

Reply:
This statement says it:

`they are not supported in staging and can only be added to the live site`

This means that the mentioned content types can't be deleted in the live environment; instead, they can only be added. This limitation is present in other types, and not just custom fields. 

Also, saying that the functionality **can't** do something could give the impression our product is limited, as we discussed other times before. So I went with the way it was already written and modified the text.

Reply:
I agree that saying that the functionality can't do something could give the impression our product is limited, but in this case **it really is a limitation** of our product. Imagine someone adding the custom field and not being able to delete it anymore, they would certainly get lost on how to proceed...

I would still stick with something like this: `Note: Since comments and ratings are meant for the end user, they are not supported in staging and can only be added to the live site. Once published, custom fields do not get deleted in remote live.`